generator/test_case_generator.go

```diff
@@ -211,9 +207,6 @@ var testTypeToTestConfig = map[string][]testConfig{
 			targets: map[string]map[string]struct{}{"arc": {"amd64": {}}},
 		},
 		{testDir: "./test/fluent", terraformDir: "terraform/eks/daemon/fluent/bit"},
-		{testDir: "./test/app_signals", terraformDir: "terraform/eks/daemon/app_signals",
```

Comment:
I imagine there is a bunch of associated code as well to generate telemetry etc.. do we plan on removing all of that as well?

Reply:
Yes, this PR is to just disable it at least as a first step to unblock the release.

There is quite a bit of code for appsignals which is mixed in with our ec2 tests as well, so will have a separate pr for it.

Reply:
I changed the wording to "disable" instead of "remove"